components/stargate/cmd/client.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"github.com/formancehq/stack/components/stargate/internal/client"
+	"github.com/formancehq/stack/libs/go-libs/otlp/otlpmetrics"
+	"github.com/formancehq/stack/libs/go-libs/otlp/otlptraces"
+	app "github.com/formancehq/stack/libs/go-libs/service"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/fx"
+	"google.golang.org/grpc/keepalive"
+)
+
+const (
+	stargateServerURLFlag = "stargate-server-url"
+	gatewayURLFlag        = "gateway-url"
+
+	workerPoolMaxWorkersFlag = "worker-pool-max-worker"
+	workerPoolMaxTasksFlag   = "worker-pool-max-tasks"
+
+	ClientChanSizeFlag                = "client-chan-size"
+	HTTPClientTimeoutFlag             = "http-client-timeout"
+	HTTPClientMaxIdleConnsFlag        = "http-client-max-idle-conns"
+	HTTPClientMaxIdleConnsPerHostFlag = "http-client-max-idle-conns-per-host"
+
+	KeepAliveClientParamTimeFlag                = "keepalive-client-param-time"
+	KeepAliveClientParamTimeoutFlag             = "keepalive-client-param-timeout"
+	KeepAliveClientParamPermitWithoutStreamFlag = "keepalive-client-param-permit-without-stream"
+)
+
+func newClient() *cobra.Command {
+	return &cobra.Command{
+		Use:          "client",
+		Short:        "Launch client",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return app.New(cmd.OutOrStdout(), resolveClientOptions(
+				viper.GetViper(),
+			)...).Run(cmd.Context())
+		},
+	}
+}
+
+func resolveClientOptions(v *viper.Viper, userOptions ...fx.Option) []fx.Option {
```

Comment:
parameters not used

Reply:
@paul-nicolas Why closed?

Reply:
oops I thought you meant the grpc keepalive param struct !